### PR TITLE
Annotations for a work

### PR DESCRIPTION
**Pull request by aslagle**
This branch adds a link to get annotations for a work, and also fixes the annotation target so it's compacted jsonld.

### CODE REVIEW

File: api/routes.py [hunk 1]
```diff
@@ -202,6 +202,13 @@ def annotations():
 def annotation_detail(annotation_id):
     return app.manager.annotations.detail(annotation_id)
 
+@app.route('/annotations/<identifier_type>/<path:identifier>/', methods=['GET'])
```
Comment (aslagle) [inline api/routes.py]: I originally wanted to put this under works, but I didn't want to have to give it a data source and without one it makes the route ambiguous.

**Comment by leonardr** [comment]
This makes sense to me, except I don't understand why you compact the annotation target. Is the context not necessary because it's embedded in a larger document?

**Comment by aslagle** [comment]
Right, the context is already in the document at the top-level so it doesn't need to be in the target again. 

Since Aferdita doesn't have a library for parsing JSON-LD it's easier for her to use the compacted version for both get and post, and it's also just a smaller thing to send the client.

**Comment by leonardr** [comment]
:+1:

**Pull request merged**